src/scss/normalize.css

```diff
@@ -31,30 +31,6 @@ main {
   display: block;
 }
 
-/**
- * Correct the font size and margin on `h1` elements within `section` and
- * `article` contexts in Chrome, Firefox, and Safari.
- */
-
-h1 {
-  font-size: 2em;
-  margin: 0.67em 0;
-}
-
-/* Grouping content
-   ========================================================================== */
-
-/**
- * 1. Add the correct box sizing in Firefox.
- * 2. Show the overflow in Edge and IE.
- */
-
-hr {
-  box-sizing: content-box; /* 1 */
-  height: 0; /* 1 */
-  overflow: visible; /* 2 */
-}
-
```

Comment:
Just curious: what was the reason for this change? (Thinking ahead to any future updates of normalize we might need to pull in.)

Reply:
h1 was because header height is actually controlled by Typography right now... hr should go back. I'll fix that.